### PR TITLE
INFRA: Initialize Acceptance Tests

### LINKED ISSUE #22: INFRA: Initialize Acceptance Tests

### PR DESCRIPTION
**Pull request by NaynaMallick**
Closes #22

### CODE REVIEW

File: ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using FluentAssertions;
+using System.Threading.Tasks;
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:5]: Run code clean up

File: ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            string expectedHomeMessage =
+                "Hello! I am ArtGallery Services";
+
+            //when
+            string actualHomeMessage =
+               await this.artGalleryApiBroker.GetHomeMessageAsync();
+
+            //then
+            actualHomeMessage.Should().BeEquivalentTo(expectedHomeMessage);
+
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs]: remove extra line.

File: ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+using ArtGallery.Services.Api.Tests.Acceptance.Brokers;
+using Xunit;
+
+namespace ArtGallery.Services.Api.Tests.Acceptance.Apis.Home
+{
+    [Collection(nameof(ApiTestCollection))]
+    public partial class HomeApiTests
+    {
+        private readonly ArtGalleryApiBroker artGalleryApiBroker;
+        public HomeApiTests(ArtGalleryApiBroker artGalleryApiBroker) =>
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.cs:15]: Need a new line space.

File: ArtGallery.Services.Api.Tests.Acceptance/ArtGallery.Services.Api.Tests.Acceptance.csproj [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api.Tests.Acceptance/ArtGallery.Services.Api.Tests.Acceptance.csproj]: Remove extra line

File: ArtGallery.Services.Api.Tests.Acceptance/Brokers/ArtGalleryApiBroker.Home.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace ArtGallery.Services.Api.Tests.Acceptance.Brokers
+{
+    public partial class ArtGalleryApiBroker
+    {
+        private const string HomeRelativeUrl = "api/home";
+
+        public async ValueTask<string> GetHomeMessageAsync() =>
+            await this.apiFactoryClient.GetContentStringAsync(HomeRelativeUrl);
+
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api.Tests.Acceptance/Brokers/ArtGalleryApiBroker.Home.cs]: remove extra line.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs (hunk 1 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:5]: resolved

File: ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs (hunk 2 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs]: resolved

**Review by ShriHumrudha — APPROVED** [review]

**Review by Shrabanac — APPROVED** [review]

**Pull request merged**